lib/opentok.js

```diff
@@ -901,6 +901,52 @@ OpenTok = function (apiKey, apiSecret, env) {
     });
   };
 
+  /**
+   * Send DTMF digits to all participants in an active OpenTok session or to a specific client
+   * connected to that session.
+   *
+   * @param sessionId The session ID corresponding to the session that will receive the DTMF string.
+   *
+   * @param connectionId The connection connection ID of the client you are sending the DTMF signal to.
+   *  Leave this empty to send a DTMF signal to all clients connected to the session.
```

Comment:
Linting error (line too long).
```suggestion
   * @param connectionId The connection connection ID of the client you are sending the DTMF signal
   * to. Leave this empty to send a DTMF signal to all clients connected to the session.
```